spec.html

```diff
@@ -42485,11 +42767,61 @@ <h1>get ArrayBuffer.prototype.byteLength</h1>
         </emu-alg>
       </emu-clause>
 
+      <emu-clause id="sec-get-arraybuffer.prototype.maxbytelength">
+        <h1>get ArrayBuffer.prototype.maxByteLength</h1>
+        <p>`ArrayBuffer.prototype.maxByteLength` is an accessor property whose set accessor function is *undefined*. Its get accessor function performs the following steps:</p>
+        <emu-alg>
+          1. Let _O_ be the *this* value.
+          1. Perform ? RequireInternalSlot(_O_, [[ArrayBufferData]]).
+          1. If IsSharedArrayBuffer(_O_) is *true*, throw a *TypeError* exception.
+          1. If IsDetachedBuffer(_O_) is *true*, return *+0*<sub>𝔽</sub>.
+          1. If IsFixedLengthArrayBuffer(_O_) is *false*, then
+            1. Let _length_ be _O_.[[ArrayBufferMaxByteLength]].
+          1. Else,
+            1. Let _length_ be _O_.[[ArrayBufferByteLength]].
+          1. Return 𝔽(_length_).
+        </emu-alg>
+      </emu-clause>
+
+      <emu-clause id="sec-get-arraybuffer.prototype.resizable">
+        <h1>get ArrayBuffer.prototype.resizable</h1>
+        <p>`ArrayBuffer.prototype.resizable` is an accessor property whose set accessor function is *undefined*. Its get accessor function performs the following steps:</p>
+        <emu-alg>
+          1. Let _O_ be the *this* value.
+          1. Perform ? RequireInternalSlot(_O_, [[ArrayBufferData]]).
+          1. If IsSharedArrayBuffer(_O_) is *true*, throw a *TypeError* exception.
+          1. If IsFixedLengthArrayBuffer(_O_) is *false* return *true*; otherwise return *false*.
+        </emu-alg>
+      </emu-clause>
+
       <emu-clause id="sec-arraybuffer.prototype.constructor">
         <h1>ArrayBuffer.prototype.constructor</h1>
         <p>The initial value of `ArrayBuffer.prototype.constructor` is %ArrayBuffer%.</p>
       </emu-clause>
 
+      <emu-clause id="sec-arraybuffer.prototype.resize">
+        <h1>ArrayBuffer.prototype.resize ( _newLength_ )</h1>
+        <p>The following steps are taken:</p>
+        <emu-alg>
+          1. Let _O_ be the *this* value.
+          1. Perform ? RequireInternalSlot(_O_, [[ArrayBufferMaxByteLength]]).
+          1. If IsSharedArrayBuffer(_O_) is *true*, throw a *TypeError* exception.
+          1. Let _newByteLength_ be ? ToIntegerOrInfinity(_newLength_).
```

Comment:
```suggestion
          1. Let _newByteLength_ be ? ToIndex(_newLength_).
```

Because I think `ToIntegerOrInfinity` is normally used when negative values are also allowed. And using `ToIndex` also matches [ArrayBufferCopyAndDetach](https://tc39.es/proposal-arraybuffer-transfer/#sec-arraybuffercopyanddetach).

Reply:
Agreed.